test/test.js

```diff
@@ -0,0 +1,27 @@
+const expect = require('chai').expect;
+const spawnSync = require('child_process').spawnSync;
+
+describe('Test help for each function', () => {
+    
```

Comment:
Next time remove all these newlines.
(We should have a linter fail for these extra newlines everywhere).